Xamarin.PropertyEditing.Mac/Controls/Custom/BasePopOverControl.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using AppKit;
+using CoreGraphics;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class BasePopOverControl : NSView
+	{
+		const int DefaultIconButtonSize = 24;
+
+		public BasePopOverControl (string title, string imageNamed)
+		{
+			if (title == null)
+				throw new ArgumentNullException (nameof (title));
+			if (imageNamed == null)
+				throw new ArgumentNullException (nameof (imageNamed));
+
+			Frame = new CGRect (CGPoint.Empty, new CGSize (320, 180));
```

Comment:
Lots of hard-coded constants in there. Are these considered defaults that derived controls will override?

Reply:
Yes, the Frame will change dependent on each derived View. I could have left the Frame as 0s, but want devs to at least see something once they fire up their derived View.